DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "## Fallacies and Manipulation\n",
+    "\n",
+    "<!-- cite: https://www.datasciencecentral.com/profiles/blogs/data-fallacies-to-avoid-an-illustrated-collection-of-mistakes -->\n",
+    "\n",
+    "The first step in being a conscious member of the data community is to identify and refute bad data appropriately. There are many resources for the many, varied ways to manipulate data, but we'll look at only the most common and most relevant ones here.\n",
+    "\n",
+    "#### Confirmation Bias\n",
+    "\n",
+    "The biggest consequence (good or bad) of the information age, is that data is available at the click of a mouse or wake-word in almost everybody's homes. It certainly is good for us to have access to more, but that means that we now have to be aware of confirmation bias as we interpret this data. If you're looking for the answer to a question that you don't have an opinion on, even slight differences in phrasing will affect the content of your results. Take Coffee and Acne for example, in a simple Google search question we see very different results based on the choice of cause/cure:\n",
+    " \n",
+    " Cure            |  Cause\n",
+    ":-------------------------:|:-------------------------:\n",
+    "![](https://i.imgur.com/z1FkMNB.png)  |  ![](https://i.imgur.com/qIPZvc2.png)\n",
+    "\n",
+    "We see that even in a case where you're not trying to put your bias into the search, that your results can be skewed. This can be resolved by looking at aggregates of data instead of single-sources. This makes way for \"pop science\" where a study finds that X causes/cures Y, where both X and Y are common conditions. We see more how these unfortunate statistical anomalies become daytime news stories here.\n",
+    "\n",
+    "\n",
+    "#### Data Dredging\n",
+    "\n",
+    "Data dredging is the effect when intentional or not, you have too many variables or opportunities for correlation for a single dataset. There is a famous example of this, when the website FiveThirtyEight reported on the specific field of nutrition, as the data available has issues already. They noted the specific issues of how they generate \"huge data sets with many, many variables,\" which are the most susceptible data dredging. [The article](https://fivethirtyeight.com/features/you-cant-trust-what-you-read-about-nutrition/) finds a few particularly remarkable relationships, where they found the following links from the respondent data:\n",
```

Comment:
- Would reword to: Data dredging is the effect when you have too many variables or opportunities (whether intentional or not) for correlation for a single dataset. 
- Would condense to: A famous example of this is when the website FiveThirtyEight reported on the specific field of nutrition, as the data available has issues already.